zenml/core/components/sequencer/utils.py

```diff
@@ -0,0 +1,65 @@
+#  Copyright (c) maiot GmbH 2021. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+
+
+import apache_beam as beam
+import tensorflow as tf
+
+
+class ToDataframe(beam.DoFn):
```

Comment:
Change to `ConvertToDataframe` for style consistency?

Reply:
I am on it!

Reply:
fixed it